feature-account-api/src/main/java/io/novafoundation/nova/feature_account_api/presenatation/account/AddressDisplayUseCase.kt

```diff
@@ -14,7 +15,7 @@ class AddressDisplayUseCase(
     class Identifier(private val addressToName: Map<String, String?>) {
 
         fun nameOrAddress(address: String): String {
-            return addressToName[address] ?: address
+            return addressToName[address] ?: address.toShortAddressFormat()
```

Comment:
We generally want to display as much of address content as possible. Leavinf only 4 symbols from each side potentially opens an attack vector where scammer mimics an address user frequently interacts with by finding another address which has similar prefix and suffix